backend/rhapp_aws_s3-main/app.py

```diff
@@ -0,0 +1,55 @@
+from boto3 import client, resource
+from credentials import ACCESS_KEY, SECRET_ACCESS_KEY
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import io
+from PIL import Image, ImageGrab
+
+user = client(
+    's3',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_ACCESS_KEY,
+    region_name='ap-southeast-1'
+)
+
+res = resource(
+    's3',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_ACCESS_KEY,
+    region_name='ap-southeast-1'
+)
+
+bucket = res.Bucket('rhapp-picture-bucket')
+
+#create
+def create(file):
+    obj = bucket.Object(file)
+    img = Image.open('stickman.png')
+    image_stream = io.BytesIO()
+    img.save(image_stream, format='png')
+    image_stream.flush()
+    obj.put(Body=image_stream.getvalue())
+
+#read
+def read(file):
+    obj = bucket.Object(file)
+    image_stream = io.BytesIO()
+    obj.download_fileobj(image_stream)
+    img = mpimg.imread(image_stream, format='jpg')
+    imgplot = plt.imshow(img)
+    plt.show()
+
+#update
+def update(file):
```

Comment:
Arugments should be oldKey, newKey, fileLocation(optional)

If fileLocation is provided, delete oldKey, create with image at fileLocation with new key

If fileLocation not provided, rename